### ISSUE TITLE
Fix remaining broken templates and instances

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-model-validation-library#68]
Production has quite a few templates that do not pass validation. See what issues are and hand patch or patch with validation script as necessary. Ditto for staging.

Goal is 100% pass rate (with daily validation as per #49).

Known current validation issues listed in these tasks:

#51
#65

**Comment by martinjoconnor** [comment]
Email from December 14th:

```
Hi Josef,

Thanks for the response. I'll address below. Let's move further discussions to:

https://github.com/metadatacenter/cedar-model-validation-library/issues/68


2. New errors got revealed after the last bug fix of the patch script and they were mostly related to the unescaped field names that I mentioned in the previous email.
I'll hand-patch the bad templates in point #2 and I’ll wait for your decision first about the model before patching the templates for point #3.

Sounds good.


a. The introduction of the ‘title’ and ‘description’ fields in the _ui object which violated the model.
c. Missing the schema:description for static template fields. Possible fix: put an empty string by default or remove it from the required list in the model.

Debra has just fixed that:

https://github.com/metadatacenter/cedar-project/issues/603

Can you patch the (hopefully few) ones that slipped through.

b. The ‘type’ field must be an array instead of a single string in the properties/@id object, such as:
 "@id": {
  "type": [
    "string",
    "null"
  ],
  "format”: "uri"
 }

d. Missing the required field for template fields with value constraints. Hint: the required field must be @id.
 “required” [
  “@id"
 ]


See comments from Marcos here: https://github.com/metadatacenter/cedar-project/issues/612

Basically, we discovered that JSON-LD does not allow null as a possible value for an @id field - so "@id": {  "type": "string" } is correct and the required array should now require @id because it is left out in the case where there is not value.

Fortunately, this means that no patching is needed - only the validation schema needs to be updated.

Will be interesting to see if there are any validation issues left after patching for 2 and 3a/c and updating validation schema for 3b/d
```

**Comment by johardi** [comment]
OK I'm going to update the model for point (b) and (d) according to your comment. And I think we still need the patching to update the old templates.
